contrib/curl/premake5.lua

```diff
@@ -6,17 +6,16 @@ project "curl-lib"
 	defines     { "BUILDING_LIBCURL", "CURL_STATICLIB", "HTTP_ONLY" }
 	warnings    "off"
 
-	if not _OPTIONS["no-zlib"] then
-		defines     { 'USE_ZLIB' }
-		includedirs { '../zlib' }
-	end
-
 	files
 	{
 		"**.h",
 		"**.c"
 	}
 
+	filter { "options:not zlib-src=none" }
+		defines     { 'USE_ZLIB' }
+		includedirs { '../zlib' }
```

Comment:
That `includedirs` is only for non-system

Reply:
Nice find, I'll fix that up now.

Reply:
That should be fixed up now, only pulls it in when using `contrib`.